js/components/controlBar.js

```diff
@@ -188,6 +188,13 @@ var ControlBar = React.createClass({
     this.removeHighlight({target: ReactDOM.findDOMNode(this.refs.volumeIcon)});
   },
 
+  handleWatermarkClick: function() {
+    var watermarkClickUrl = this.props.skinConfig.controlBar.watermark.clickUrl;
+    if (watermarkClickUrl){
+      window.open(watermarkClickUrl);
+    }
```

Comment:
should this default to open in new window?

window.open(watermarkClickUrl, '_blank');